CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 4.1.1
```

Comment:
Let's make this a minor update and explicitly mention the change in logstash version requirements